openpype/settings/defaults/project_settings/substancepainter.json

```
@@ -0,0 +1,13 @@
+{
+    "imageio": {
+        "ocio_config": {
+            "enabled": true,
+            "filepath": []
+        },
+        "file_rules": {
+            "enabled": true,
+            "rules": {}
+        }
+    },
+    "shelves": {}
+}
```

We've changed it to have empty line at the end
```suggestion
}

```